src/core/CommandsTest.js

```diff
@@ -0,0 +1,106 @@
+const fs = require("fs");
+
+global.typeVariable = {
+	INTEGER: {
+		type: "number",
+		check: (v) => typeof v === typeVariable.INTEGER.type
+	},
+	MENTION: {
+		type: "mention",
+		check: (v) => isAMention(v)
+	},
+	EMOJI: {
+		type: "emoji",
+		check: (v) => isAnEmoji(v)
+		},
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2.](https://app.codacy.com/gh/DraftBot-A-Discord-Adventure/DraftBot/pullRequest?prid=7613090)
```suggestion
	},
```